grails-app/controllers/org/pih/warehouse/api/DashboardApiController.groovy

```diff
@@ -106,7 +104,7 @@ class DashboardApiController {
     }
 
     def getFillRateDestinations() {
-        Location location = Location.get(params.locationId?:session.warehouse.id)
+        Location location = Location.get(params.locationId ?: session.warehouse.id)
```

Comment:
@awalkowiak How do we feel about these automatic formatting changes?

Reply:
I am ok with it